lib/rules/array-bracket-spacing.js

```diff
@@ -130,19 +85,19 @@ module.exports = function(context) {
 
         if (isSameLine(first, second)) {
             if (openingBracketMustBeSpaced && !isSpaced(first, second)) {
-                reportRequiredBeginningSpace(node, first);
+                reportError(node, context.errors.spaceBefore, first);
```

Comment:
I'm not sure there's a straight line between translations and changing what context is meant to do. I'd need to see a more fully fleshed out proposal for i18n.